### PR TITLE
BLUEBUTTON-972: Added SAMPLE_MCT data set

### PR DESCRIPTION
**Pull request by karlmdavis**
Added a `SAMPLE_MCT` data set that is approved for public use, and intended to support development and testing for our MCT friends in our DPR environment.

https://jira.cms.gov/browse/BLUEBUTTON-972

### CODE REVIEW

**Comment by karlmdavis** [comment]
Note: I added a `RifLoaderIT.loadSampleMctData()` test case for this new data (coming in a separate set of PRs). As of my last commit, that test case passes.

File: bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+INSERT|-202|42|999|35487|01-JAN-1910|2|1|0|0|0|10|0|0|MBP000202A|SMITH|STEPHEN|E|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|H0104|H0104|H0104|H0104|0|0|0|0|0|0|0|0|012|012|012|012|||||||||0|0|0|0|||||||||||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||C|C|C|C|C|C|C|C|C|C|C|C|02|02|02|02|00|00|00|00|00|00|00|00|10|10|10|10|00|00|00|00|00|00|00|00
+INSERT|-203|05|999|72653|02-FEB-1910|1|1|0|0|0|10|0|0|MBP000203A|JOHNSON|MICHAEL|V|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||S4802|S4802|S4802|S4802|0|0|0|0|0|0|0|0|073|073|073|073|||||||||0|0|0|0|||||||||C|C|C|C|C|C|C|C|C|C|C|C|NA|NA|NA|NA|00|00|00|00|00|00|00|00|01|01|01|01|00|00|00|00|00|00|00|00
```
Comment (dho937) [inline bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt]: PTD_SGMT_***_ID is stored in the production database as '000' whereas you have '0' as the sample data.  I think we should emulate how the data is actually coming to us from CCW so issues don't arise when MCT's code is migrated to production.

File: bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+INSERT|-204|52|999|63400|24-FEB-1910|2|1|0|0|0|10|0|0|MBP000204A|JONES|PETER|L|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|H0710|H0710|H0710|H0710|0|0|0|0|0|0|0|0|026|026|026|026|||||||||0|0|0|0|||||||||||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||C|C|C|C|C|C|C|C|C|C|C|C|00|00|00|00|00|00|00|00|00|00|00|00|02|02|02|02|00|00|00|00|00|00|00|00
+INSERT|-207|36|999|46250|04-APR-1920|1|1|0|0|0|10|0|0|MBP000207A|WRIGHT|CHRISTOPHER|D|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2013|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|H5619|H5619|H5619|H5619|0|0|0|0|0|0|0|0|055|055|055|055|||||||||0|0|0|0|||||||||||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||C|C|C|C|C|C|C|C|C|C|C|C|09|09|09|09|00|00|00|00|00|00|00|00|06|06|06|06|00|00|00|00|00|00|00|00
```
Comment (dho937) [inline bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt]: PTC_PLAN_TYPE_***_CD is stored in the production database as '00' whereas you have '0' as the sample data. I think we should emulate how the data is actually coming to us from CCW so issues don't arise when MCT's code is migrated to production.

File: bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt (hunk 1 above)
Comment (karlmdavis) [reply bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt]: Great catch: thanks so much! I've fixed this in a new commit.

@dho937 Can you take a second look at it now?

File: bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt (hunk 2 above)
Comment (karlmdavis) [reply bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-beneficiaries.txt]: Great catch: thanks so much! I've fixed this in a new commit.

@dho937 Can you take a second look at it now?

**Comment by glovercloud** [comment]
Hi Karl,

Can you take me off the mailing list?  It’s glovercloud@comcast.net.

Thanks,
Brian



Brian

**Pull request merged**